pom.xml

```diff
@@ -22,7 +22,7 @@
 
   <groupId>io.cdap.delta</groupId>
   <artifactId>bigquery-delta-plugins</artifactId>
-  <version>0.2.1-SNAPSHOT</version>
+  <version>0.2.2-SNAPSHOT</version>
```

Comment:
why change this? We did not release 0.2.1 version (CDAP 6.3.1) right?

Reply:
ah, I see. I thought we released 0.2.1 for CDAP 6.3. Then let me create a new release branch